src/theme/CodeBlock/index.js

```diff
@@ -1,181 +0,0 @@
-/**
```

Comment:
We don’t need to do this manually now since Docusaurus provides the copy code button now